package.json

```diff
@@ -1,183 +0,0 @@
-{
```

Comment:
It seems for some reason this whole file has been deleted, can you please restore it?

(I've had this happen to me once too, I'm not sure what causes it)